_meta/beat.yml

```diff
@@ -374,8 +393,14 @@ output.elasticsearch:
       when.contains:
         processor.event: "onboarding"
 
-  # Optional ingest node pipeline. By default no pipeline will be used.
-  #pipeline: ""
+  # Ingest node pipelines are applied when writing documents to Elasticsearch.
```

Comment:
reading this i am not clear if i need to uncomment `  #pipelines: - pipeline: "apm_user_agent"` or not...